### PR TITLE
Add charidotella + deps

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Hi @conda-forge/help-python-c - this is ready for review. Many thanks!

File: recipes/colourtime/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+c_stdlib_version:              # [osx and x86_64]
+  - 10.12                      # [osx and x86_64]
```
Comment (xhochy) [inline recipes/colourtime/conda_build_config.yaml:2]: This is lower than the global pinning, simply remove this file.

File: recipes/event-stream/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools
+    - wheel
+    - numpy
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```
Comment (xhochy) [inline recipes/event-stream/meta.yaml:30]: `numpy` has a run_exports and thus this should no longer be used.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by xhochy** [comment]
For `osx-64` to fix we may need to wait 15-60min. I have pushed fixes to the build scripts (no need to rebase)

**Comment by Tobias-Fischer** [comment]
Thanks @xhochy  for the quick review!

**Comment by xhochy** [comment]
The correct fix for OSX will be https://github.com/conda-forge/staged-recipes/pull/26449

**Comment by Tobias-Fischer** [comment]
Thanks @xhochy - this should be ready

**Review by xhochy — APPROVED** [review]

**Pull request merged**